backend/protocol_rpc/endpoints.py

```diff
@@ -127,7 +128,7 @@ def create_validator(
     # fallback for default provider
     # TODO: only accept all or none of the config fields
     llm_provider = None
-    if not (config and plugin and plugin_config):
+    if not (config is None and plugin is None and plugin_config is None):
```

Comment:
@denishacquin this is not what the comment says. This condition is equal to:
if config != None or plugin != None or plugin_config != None

Reply:
fixed this, thanks for the spot!